includes/class-sensei-grading-user-quiz.php

```diff
@@ -215,6 +215,26 @@ public function display() {
 						break;
 				}
 
+				/**
```

Comment:
As we consider hooks part of Sensei's public API, we need to document this a bit better before merging:
- The 'Other question type filters.' comment does not offer much information.
- A better description for the filter would be 'Filters various values which are displayed in the grading admin page' or something similar. Then add documentation for each key of the array of the first value.  Have a look [here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-1-parameters-that-are-arrays) on how to document arrays.
- In the documentation, you should name the filtered variable even if it is null by default (e.g. $display_values).
- When you document `apply_filters` you should always supply a @return tag otherwise the doc-generating script will think that it's an action.
- Variable types should be inside curly brackets.
- You can have a look [here](https://github.com/Automattic/sensei/blob/88bd5b37343e44f874db11f077233db63ae7b8f8/includes/sensei-functions.php#L310) for an example.

Reply:
Thanks for the detailed instructions @gkaragia !